src/prelude/internals.mo

```diff
@@ -412,9 +414,10 @@ func @install_actor_helper(
       case (#new settings) {
```

Comment:
I wouldn't say it is a breaking change, it is just an _observable_ change. We should support the IC features by default, and offer `#*NoTrace` alternatives for those which prefer secrecy.

Reply:
`#newIncognito`, etc., or `#new : Bool`? We can decide later, when we implement this. (The latter would break current code forcing people to adapt, but suffers from boolean blindness.)